### ISSUE TITLE
ivectorTest error. Seems to be an indexing issue 

### ISSUE THREAD

**Issue by idnavid** [cyu0913/CRSS-SpkrDiar#1]
ERROR (ivectorTest:SubMatrix():kaldi-matrix.cc:1466)
This problem doesn't always happen.

**Comment by idnavid** [comment]
It doesn't seem to have a problem when I test the i-vectors for each file separately. 
With the new test.sh script, it should be fine. 
The problem only occurs when I try to pool i-vectors from multiple files together, which is not something we'll need to do.

**Comment by cyu0913** [comment]
Yeah, this script was only for checking and debugging purpose, so it's not necessary to support multiple files.

**Comment by idnavid** [comment]
Don't worry about that Chengzhu, I added that for myself, so I remember to fix it.
